cmd/clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -98,14 +98,21 @@ mkdir -p ${OUTPUT_DIR}
 
 # all variables used for yaml generation
 
-CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
-SERVICE_CIDR=${SERVICE_CIDR:-100.64.0.0/13}
-CLUSTER_CIDR=${CLUSTER_CIDR:-100.96.0.0/11}
-
-VSPHERE_USER=${VSPHERE_USER:-}
-VSPHERE_PASSWORD=${VSPHERE_PASSWORD:-}
-VSPHERE_SERVER=${VSPHERE_SERVER:-}
-VSPHERE_MANAGER_IMG=${VSPHERE_MANAGER_IMG:-}
+export CLUSTER_NAME=${CLUSTER_NAME:-vsphere-cluster}
```

Comment:
Hi @andrewsykim,

Maybe CAPA did this, and it looks like we used to, but if we default to a cluster name, will this cause problems?

Reply:
I'm assuming this is OK because it's an "example" run through but I have no problems changing this

Reply:
Stupid question -- this isn't the same template used to create the management cluster? I thought these templates were used for both?

Reply:
Yup, but I'm assuming someone that wants to stand up a production management cluster would set `CLUSTER_NAME` to something that is more reasonable. `vsphere-cluster` as a default for `cmd/clusterctl/examples` seems okay to me

Reply:
fwiw https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/cmd/clusterctl/examples/aws/generate-yaml.sh#L27

Reply:
Reverting to something like `vsphere-mgmt-test1` might be better so users know to not use the example cluster for production

Reply:
All fair points. I guess I just look at this with fresh eyes instead of improving what we have. Looks good, thanks.